src/cortexpy/tesserae.py

```diff
@@ -369,12 +365,12 @@ def __initialization(self, query, panel, lsize_l):
                     self.vt_i[seq][pos_seq][1] = self.lpiI - lsize_l + self.lsi[convert[query[0]]]
 
                     if pos_seq > 0:
-                        self.vt_d[seq][pos_seq][1] = self.vt_m[seq][pos_seq - 1][1] + self.ldel
-                        self.tb_d[seq][pos_seq][1] = seq * 10 + 1 + (pos_seq - 1) / self.tb_divisor
-
-                        if self.vt_d[seq][pos_seq - 1][1] + self.leps > self.vt_d[seq][pos_seq][1]:
-                            self.vt_d[seq][pos_seq][1] = self.vt_d[seq][pos_seq - 1][1] + self.leps
-                            self.tb_d[seq][pos_seq][1] = seq * 10 + 3 + (pos_seq - 1) / self.tb_divisor
+                        vt_d_1, tb_d_1 = max([
```

Comment:
I don't think the brackets (`[` and `]`) are necessary here?

Reply:
Agreed - max(...) has a function prototype that takes multiple arguments, removing the need to prepack everything into a tuple first.